src/pymoca/ast.py

```diff
@@ -220,69 +226,88 @@ def concatenate(self, arg: 'ComponentRef') -> 'ComponentRef':
 class Expression(Node):
     def __init__(self, **kwargs):
         self.operator = None  # type: Union[str, ComponentRef]
-        self.operands = []  # type: List[Union[Expression, Primary, ComponentRef, Array, IfExpression]]
+        self.operands = (
+            []
+        )  # type: List[Union[Expression, Primary, ComponentRef, Array, IfExpression]]
```

Comment:
Why did it break this line (was only 87 long)?

Reply:
Perhaps it is so that you can easily see in a diff changes to only type hints change vs changes in the functioning code?

Reply:
I guess, we should probably make type annotations proper Python 3 style as well, not as comments. There's a whole bunch of legacy stuff in the code base still

Reply:
Marking this as resolved. Did you record your thought somewhere (maybe an Issue?) about changing to Python 3 type annotations?